client/blocks/post-share/nudges.jsx

```diff
@@ -7,48 +7,71 @@ import React from 'react';
  * Internal dependencies
  */
 import Banner from 'components/banner';
-import { PLAN_BUSINESS } from 'lib/plans/constants';
+import { PLAN_PREMIUM, PLAN_JETPACK_PREMIUM } from 'lib/plans/constants';
 import formatCurrency from 'lib/format-currency';
 
 export const UpgradeToPremiumNudge = props => {
 	const {
-		businessDiscountedRawPrice,
-		businessRawPrice,
+		premiumPrice,
+		jetpackPremiumPrice,
 		translate,
 		userCurrency,
+		isJetpack,
 	} = props;
 
+	let price, featureList, proposedPlan;
+	if ( isJetpack ) {
+		price = jetpackPremiumPrice;
+		featureList = [
+			translate( 'Schedule your social messages in advance.' ),
+			translate( 'Easy monetization options' ),
+			translate( 'VideoPress support' ),
+			translate( 'Daily Malware Scanning' ),
+		];
+		proposedPlan = PLAN_JETPACK_PREMIUM;
+	} else {
+		price = premiumPrice;
+		featureList = [
+			translate( 'Schedule your social messages in advance.' ),
+			translate( 'Remove all advertising from your site.' ),
+			translate( 'Enjoy live chat support.' ),
+			translate( 'Easy monetization options' ),
+			translate( 'Unlimited premium themes.' ),
+		];
+		proposedPlan = PLAN_PREMIUM;
+	}
+
 	return (
 		<Banner
 			className="post-share__actions-list-upgrade-nudge"
 			callToAction={
 				translate( 'Upgrade for %s', {
-					args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+					args: formatCurrency( price, userCurrency ),
 					comment: '%s will be replaced by a formatted price, i.e $9.99'
 				} )
 			}
-			list={ [
-				translate( 'Schedule your social messages in advance.' ),
-				translate( 'Remove all advertising from your site.' ),
-				translate( 'Enjoy live chat support.' ),
-				translate( 'Ability to add features through external plugins.' ),
-				translate( 'Access to thousands of themes.' ),
-			] }
-			plan={ PLAN_BUSINESS }
-			title={ translate( 'Upgrade to a Business Plan!' ) } />
+			list={ featureList }
+			plan={ proposedPlan }
+			title={ translate( 'Upgrade to a Premium Plan!' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Upgrade to a Premium Plan and Enable VideoPress' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/publicize/enable-republicize-for-jp-personal)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).